app/components/chips/base_chip.tsx

```diff
@@ -11,17 +11,17 @@ import {nonBreakingString} from '@utils/strings';
 import {changeOpacity, makeStyleSheetFromTheme} from '@utils/theme';
 import {typography} from '@utils/typography';
 
+import {CHIP_BOTTOM_MARGIN, CHIP_HEIGHT} from './constants';
+
 type SelectedChipProps = {
-    id: string;
-    text: string;
-    extra?: React.ReactNode;
-    onRemove: (id: string) => void;
+    onPress: () => void;
     testID?: string;
-    containerStyle?: StyleProp<ViewStyle>;
+    showXButton?: boolean;
+    showAnimation?: boolean;
```

Comment:
Maybe this we can do something more granular like

runEnterLayoutAnimation and runExitLayoutAnimation